Update README.md

ashishsiot opened the pull request:
update readme.md 
correct spellings and grammatical miskicks.
thank you

ashishsiot committed:
Update README.md

update readme.md 
correct spellings and grammatical miskicks.
thank you

zserge commented:
May I ask you to update the README.sh instead? README.md is auto-generated from that script.

ashishsiot committed:
update draw.html

add to extra buttons to  perform action like 
- new canvas  = it is use for open new canvas in to new tab.
-  Clear All =  it is used to  clear all brush stocks and make canvas clear as new

ashishsiot committed:
Merge pull request #1 from ashishsiot/ashishsiot-patch-1-draw

update draw.html

magistau commented on README.md:
```diff
@@ -10,7 +10,7 @@ Awfice is a collection of tiny office suite apps:
 * which can be also sold as a "good for your privacy" feature
 * this project is only a half-joke, I actually use a few Awfice apps as quick scratchpads.
-* the only way to save your job is to save a HTML or send it to the printer/print to PDF.
+* The only way to save your job is to save a HTML or send it to the printer/print to PDF.
```
I don't think this one should be capital

magistau commented on draw.html:
```diff
@@ -1 +1,24 @@
-<canvas id=v><script>d=document,d.body.style.margin=0,P="onpointer",c=v.getContext`2d`,v.width=innerWidth,v.height=innerHeight,c.lineWidth=2,f=0,d[P+"down"]=e=>{f=e.pointerId+1;e.preventDefault();c.beginPath();c.moveTo(e.x,e.y)};d[P+"move"]=e=>{f==e.pointerId+1&&c.lineTo(e.x,e.y);c.stroke()},d[P+"up"]=_=>f=0</script></canvas>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+    <head>
+        <FORM>
+            <INPUT TYPE="button" onClick="history.go(0)" VALUE="Clear All">
+            <input type="button" value="New Canvas"
+            onclick="window.open('draw.html')">
+            </FORM>   
+    </head>
+<body>    
+ <canvas id=v>
+    <script>
+        d=document,d.body.style.margin=0,P="onpointer",
+        c=v.getContext`2d`,
+        v.width=innerWidth,
+        v.height=innerHeight,
+        c.lineWidth=2,f=0,
+        d[P+"down"]=e=>
+        {f=e.pointerId+1;e.preventDefault();c.beginPath();c.moveTo(e.x,e.y)};d[P+"move"]=e=>{f==e.pointerId+1&&c.lineTo(e.x,e.y);c.stroke()},
+        d[P+"up"]=_=>f=0
+    </script>
+</canvas>
+</body>
+</html>
```
This is a lot of meaningless extra bytes

magistau reviewed (changes requested):
Also change `README.sh` instead of `README.md`
